docs/ecosystem-user-event.mdx

```diff
@@ -204,6 +206,36 @@ test('delete characters within the selectedRange', () => {
   userEvent.type(input, '{backspace}good')
 
   expect(input).toHaveValue('This is a good example')
+```
+
+By default, `type` appends to the existing text. To prepend text, reset the
+element's selection range and provide the `initialSelectionStart` and
+`initialSelectionEnd` options:
+
+```jsx
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+test('prepend text', () => {
+  render(
+    <div>
+      <label htmlFor="my-input">Example:</label>
+      <input id="my-input" type="text" />
+    </div>,
+  )
+
+  const input = screen.getByLabelText(/example/i)
+  userEvent.type(element, 'World!')
```

Comment:
```suggestion
  render(<input defaultValue="World!"/>)
  const element = screen.getByRole('textbox')
```
Let's reduce the noise.
`element` instead of `input` is used as variable name in the following lines.